util/filter_bench.cc

```diff
@@ -701,7 +738,11 @@ int main(int argc, char **argv) {
         << "\n      of queries." << std::endl;
   } else {
     FilterBench b;
-    b.Go();
+    for (uint32_t i = 0; i < FLAGS_runs; ++i) {
+      b.Go();
+      FLAGS_seed += 100;
```

Comment:
Nit: it is not recommended to change the value of `FLAGS_` variables, but this is truly minor.